client/src/components/SubstraitGraph.vue

```diff
@@ -113,6 +131,20 @@ export default {
       plan_link.click();
       URL.revokeObjectURL(plan_link.href);
     },
+    async generateLink() {
```

Comment:
Currently, even if the plan is not changed and the Copy Link button is clicked, this will again save the plan and generate a new link which is unnecessary, we should avoid this.

Reply:
made the change.